src/SablierV2LockupDynamic.sol

```diff
@@ -168,6 +171,11 @@ contract SablierV2LockupDynamic is
         }
     }
 
+    /// @inheritdoc ISablierV2LockupDynamic
+    function getStreamsByUser(address recipient) external view returns (uint256[] memory) {
```

Comment:
Let's rename to `getRecipientStreamIds`

Reply:
See https://github.com/sablier-labs/v2-core/issues/644#issuecomment-1706320511